net/apinger/Makefile

```diff
@@ -46,7 +46,7 @@ define Package/apinger/description
 endef
 
 define Package/apinger/conffiles
-/etc/apinger.conf
+/etc/config/apinger
```

Comment:
Should we not add also the file `/etc/apinger.user` to conffiles? To preserve this file over a sysupgrade

Reply:
Yes, added /etc/apinger.usr to conffiles